remme/clients/pub_key.py

```diff
@@ -170,7 +171,7 @@ def get_pub_key_signature(key, sig):
             bytes.fromhex(sig),
             padding.PSS(
                 mgf=padding.MGF1(hashes.SHA512()),
-                salt_length=padding.PSS.MAX_LENGTH
+                salt_length=20
```

Comment:
Was the `padding.PSS.MAX_LENGTH` platform-dependent?

Reply:
@eugene-babichenko there is no max length on JS library.

Reply:
Salt length depends on the hash function being used. I suggest implementing that in JS library.

https://github.com/pyca/cryptography/blob/3bab819c558ead888825e33f8df3f17707ff2c20/src/cryptography/hazmat/primitives/asymmetric/padding.py#L72

Reply:
@eugene-babichenko will revert back than

Reply:
@ArtemFrantsiian will you be able to reproduce that logic?

Reply:
> @ArtemFrantsiian will you be able to reproduce that logic?

Reproduce into JS library